hod_threshold:
  vel: 50
  acc: 10
  jerk: 200
  snap: 500
cost_function:
  vel_deviation_weight: 0.7
  obstacle_dist_weight: 0.3
command_stop:
  collision_radius: 0.4
  stopping_radius: 1
  stopping_angle: 0.6 # threshold for angle offset of nearest obstacle from current heading 
  free_ratio_thresh: 0.3   # threshold for ratio of free points to sampled points
  delta_free_thresh: 0.02 # threshold for delta free points ratio from previous sampling
  delta_count_thresh: 4    # number of consecutive samples below delta_free_thresh until stop commanded
  dist_weight: 0.9 # Scale the angle cost when calculating stop cost
  bias: 1.2 # Bias when calculating stop cost
  vel_weight: 0.2 # Scale velocity cost when calculating stop cost
escape_point_generation:
  grid_length: 1 # Scale of sample space length, which is based on velocity norm
  grid_width: 1 # Scale sample space width, TODO IMPROVE THIS COMMENT
  sample_length: 0.3 # discretization interval of points to sample
  sample_width: 0.3  # discretization interval of points to sample
  sample_height: 0.3
sample_method: 2 # 0=none, 1=weighted random, 2=stratified, 3=best n
sample_num: 100 # number of points to keep after sampling, to generate trajectories to
stratified_sample:
  sample_fraction: [0.01, 0.09, 0.4, 0.5] # percent of points for each strata, ordered highest to lowest strata. Should sum to 1.
  sample_num: [10, 40, 30, 20] # max number of points to sample from each strata
compute_thresh: 0.01 # do not attempt computing escape points if vehicle velocity is below this threshold
record: 0 # 0 for should not log, 1 otherwise